libs/wallet/src/web3-react/connectors/TrezorConnector/index.ts

```diff
@@ -143,7 +143,7 @@ export class TrezorConnector extends Connector {
 
     this.customProvider = customProvider
 
-    const network = await customProvider.getNetwork()
```

Comment:
should we make a util since we are repeating this many places. Also would help later with the refactor to find all uses of it

`getChainIdEagerly(provider)`?

Reply:
https://github.com/cowprotocol/cowswap/pull/5354/commits/5f717fc33053c7ee54879510e3dc7fef3b769338